DESCRIPTION

```diff
@@ -21,10 +21,7 @@ Imports:
 Suggests:
     highr,
     Cairo,
-    testthat,
-    data.table,
-    dplyr,
-    htmlwidgets
+    testthat
 Enhances: data.table, dplyr, htmlwidgets
```

Comment:
> Then what is the difference to data.table?

[this](https://github.com/IRkernel/repr/blob/master/R/repr_matrix_df.r#L74-L76).

* `data.table` provides a class that repr can handle. therefore it **`Enhances`** `data.table`
* `dplyr` creates data.frames and matrices, and is simply used in repr’s tests because it’s convenient. → not `Enhances`